spec/classes/init_spec.rb

```diff
@@ -790,6 +792,18 @@
     end
   end
 
+  [true,'invalid'].each do |proxycommands|
```

Comment:
the iterator variable, proxycommands, should be singular as it refers to a single element of the array. Please change to 'proxycommand'.